### DISCUSSION TITLE
Readme outdated in latest version ?

### DISCUSSION THREAD

**Discussion by VanillaIce24** [Leantime/leantime#2757] [General] [answered]
Did not feel like creating a issue for this, but it looks to me like the readme (especially installation) is outdated ? Build commands not working ,because they are missing in package.json

**Comment by marcelfolaron** [comment]
What build commands are not working and what commands would be missing?
We are using a makefile to run the command and everything else is run via
npm, npx and composer.

**Comment by VanillaIce24** [comment, marked as answer]
Download latest 3.2.1. release. 


Then follow this from the readme.md 



### Local Development Installation

    Clone repository to your local server
    Create MySQL database
    Run webpack builder via make **build-dev**   
    Point your local domain to the public/ directory
    Rename config/.env.sample to config/.env
    Fill in your database credentials (username, password, host, dbname) in config/.env
    Navigate to <localdomain>/install
    Follow instructions to install database and user account

### Two things that seem different as described:

1. In the latest release I have to run **composer install.**
2. The make build-dev command does not work, do you mean **npx make build dev** ?

**Reply by marcelfolaron** [reply]
What OS are you using? 
The make command is `make build-dev` but it requires you to have `make` installed. On windows you can get that via git-bash and other means. It comes standard on mac and linux.
